packages/wrangler/src/api/integrations/bindings/index.ts

```diff
@@ -41,6 +41,11 @@ export type BindingsProxy<Bindings = Record<string, unknown>> = {
 	 * Object containing the various proxies
 	 */
 	bindings: Bindings;
+	/**
+	 * Mock of the context object that Workers received in their request handler, all the object's methods are no-op
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	cf: Record<string, any>;
```

Comment:
PS: I wanted to use `IncomingRequestCfProperties` here but I've been having issues with the `@microsoft/api-extractor` package (it seems not to cope with `workers-types` for some reason 😕)

I would go with this simple type for now and look into improving the types later on (also as I want to get proper types for #4792 anyways)

But I don't mind spending more time investigating the `api-extractor` problem for this PR if that's preferred 🙂

Reply:
how about we make this `Record<string, string>` initially and later replace it with `IncomingRequestCfProperties`? That way you'll be widening the type which is a non-breaking change.

If you go with `any`, you'll be forced to narrow the type in the future which is breaking.